### ISSUE TITLE
timeSeriesRequest fails on NA vector

### ISSUE THREAD

**Issue by tega** [CharlesCara/DatastreamDSWS2R#4]
When the request should return a vector of NA then the request fails:
xtsData <- mydsws$timeSeriesRequest(instrument = "KO:SGL",
                       datatype = "AX",
                       startDate = "2018-05-07",
                       endDate = "2018-05-09",
                       frequency = "D")

On the contrary, if a numeric value is included in the result then the request 
returns correctly:
xtsData <- mydsws$timeSeriesRequest(instrument = "KO:SGL",
                       datatype = "AX",
                       startDate = "2018-05-03",
                       endDate = "2018-05-07",
                       frequency = "D")

**Comment by CharlesCara** [comment]
Hi Tega.  Thanks for raising this.  With the first query, I am getting the error:
```
Error in xts::xts(matrix(NA, nrow = 1L, ncol = myNumInstrument), order.by = endDate) : 
  order.by requires an appropriate time-based object

```
The actual response from the Datastream Server is 
```
{"DataResponses":[{"AdditionalResponses":null,"DataTypeNames":null,"DataTypeValues":[{"DataType":"AX","SymbolValues":[{"Currency":null,"Symbol":"KO:SGL","Type":0,"Value":"$$ER: 0904,NO DATA AVAILABLE"}]}],"Dates":null,"SymbolNames":null,"Tag":null}],"Properties":null}
```
So Datastream is not returning any dates to put into an xts.  We do not have any other requests to help provide an index.  

My philosophy has been to do as little manipulation as possible to the Datastream data.  So rather than guessing a vector of dates, in this case I think we should return an zero row xts.

**Comment by tega** [comment]
Dear Charles,

I fully agree with you!

**Comment by CharlesCara** [comment]
Fixed in version 1.3.2, which is now on Github
